src/main/java/com/iridium/iridiumskyblock/commands/BankCommand.java

```diff
@@ -2,31 +2,39 @@
 
 import com.iridium.iridiumcore.utils.StringUtils;
 import com.iridium.iridiumskyblock.IridiumSkyblock;
-import com.iridium.iridiumskyblock.bank.BankItem;
+import com.iridium.iridiumskyblock.commands.subcommands.BankGive;
+import com.iridium.iridiumskyblock.commands.subcommands.BankRemove;
+import com.iridium.iridiumskyblock.commands.subcommands.BankSet;
 import com.iridium.iridiumskyblock.database.Island;
-import com.iridium.iridiumskyblock.database.IslandBank;
 import com.iridium.iridiumskyblock.database.User;
 import com.iridium.iridiumskyblock.gui.BankGUI;
-import org.bukkit.Bukkit;
 import org.bukkit.command.CommandSender;
-import org.bukkit.entity.HumanEntity;
 import org.bukkit.entity.Player;
 
+import java.util.Arrays;
 import java.util.Collections;
 import java.util.List;
 import java.util.Optional;
 import java.util.stream.Collectors;
+import java.util.stream.Stream;
 
 /**
  * Command which opens the Island bank GUI.
  */
 public class BankCommand extends Command {
 
+    public BankGive bankGive;
+    public BankSet bankSet;
+    public BankRemove bankRemove;
```

Comment:
You can't configure these commands, can you?

Reply:
yep you can, the bank command gets serialized, which includes all objects inside it

Reply:
Gets created like this https://pastebin.com/p12PFBqp